agent/index.js

```diff
@@ -141,6 +148,19 @@ class Agent {
 		});
 	}
 
+	_runOnce(Job) {
+		if (!runOnce) {
```

Comment:
Where this variable declared in this scope?

Reply:
done